updater/reports/ReportOrgHistory.py

```diff
@@ -0,0 +1,55 @@
+from .ReportDaily import *
+
+# Lists the total number of organizations
+class ReportOrgHistory(ReportDaily):
+	def name(self):
+		return "org-history"
+
+	def updateDailyData(self):
+		newHeader, newData = self.parseData(
+			self.executeQuery(self.query())
+		)
+		self.header = newHeader if newHeader else self.header
+		self.data.extend(newData)
+		self.truncateData(self.timeRangeTotal())
+		self.sortDataByDate()
+
+	# Collects the number of organizations
+	def subquery(self, userType):
+		query = '''
+			SELECT
+				COUNT(*) AS count
+			FROM
+				users
+			WHERE
+				users.type = "Organization"
+			AND
+				users.login <> "github-enterprise"
+			'''
+
+		return query
+
+	# Collects the number of teams
+	def totalTeamsQuery(self, userType):
+		query = '''
+			SELECT
+				COUNT(*) AS count
+			FROM
+				teams
+			'''
```

Comment:
Isn't that the total number of teams? Don't we want to know the number of teams per org here?
I dunno if we should mix that into this repo. @pluehne ?

Reply:
As I hit "send", I realized you really want to count the total number of teams here 😄